src/parser.rs

```diff
@@ -543,7 +543,7 @@ mod tests {
                     assert_eq!(let_statement.mutable, test_case.mutable);
                     // TODO: Check some property of the expression.
                 }
-                _ => panic!("Expected a let statement"),
+                Statement::Return(_) => panic!("Expected a let statement"),
```

Comment:
Q: What's the motivation for this change? I'm not sure how it's better as adding new statements will mean this needs updating needlessly. I'm sure I'm missing something.

Reply:
Making the pattern match explicit and exhaustive means code takes into account all possible values. Once there are more than two kinds of statement, clippy will stop complaining I think